src/backend/distributed/executor/multi_utility.c

```diff
@@ -835,6 +857,53 @@ ErrorIfDistributedRenameStmt(RenameStmt *renameStatement)
 
 
 /*
+ * CreateLocalTable gets DDL commands from the remote node for the given
+ * relation. Then, it creates the local relation as temporary and on commit drop.
+ */
+static void
+CreateLocalTable(char *relationName, char *nodeName, int32 nodePort)
+{
+	List *ddlCommandList = NIL;
+	ListCell *ddlCommandCell = NULL;
+
+	/* fetch the ddl commands needed to create the table */
+	StringInfo tableNameStringInfo = makeStringInfo();
+	appendStringInfoString(tableNameStringInfo, relationName);
```

Comment:
This doesn't include the schema name.

So tables with schema names produce the following error:

    postgres=# COPY test.customer_reviews FROM '/home/marco/citus/customer_reviews_1998.csv' WITH (FORMAT CSV, MASTER_HOST localhost);
    WARNING:  could not receive query results from localhost:5432
    DETAIL:  Client error: relation "customer_reviews" does not exist
    ERROR:  could not run copy from the worker node